content/topics/solo-learn/introduction-to-sql/4-data-analysis/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+content_type: topic
+flavours:
+- none
+prerequisites:
+  hard:
+  - topics/solo-learn/introduction-to-sql/3-query-techniques
+  soft: []
+ready: true
+tags:
+- sql
+- solo-learn
+title: Solo Learn - Introduction to SQL - 4 Data Analysis
+---

```

Comment:
This should be a project, with a link submission because it's the last module in the Sololearn course and we want to check that the learner has done everything in the intro course so far.

You can have a look at the `4-challenges` module in the _depricated course to see how it's done.